app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comments = Comment.all.order(:id).reverse
+    @post = @comment.post
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create( comment_params )
+    @comment.update( timestamp: Time.now.strftime("%b %e %Y, %l:%M%P") )
```

Comment:
Do you need line 16 here?. Timestamps are managed automatically if you use `created_at` and `updated_at` as column names